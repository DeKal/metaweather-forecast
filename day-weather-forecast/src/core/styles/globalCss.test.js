import React from 'react'
import { render } from '@testing-library/react'
import GlobalCss from 'core/styles/globalCss'

describe('globalStyle test', () => {
  test('should render correctly', () => {
    const { container } = render(<GlobalCss />)
    expect(container).toMatchSnapshot()
  })
})
