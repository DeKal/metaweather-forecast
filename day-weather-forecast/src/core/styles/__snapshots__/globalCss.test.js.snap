// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`globalStyle test should render correctly 1`] = `<div />`;
